config.yml

```diff
@@ -1,40 +1,67 @@
 free_world: world
+# world imprisoned players respawn to once freed
 free_respawn: true
+# if set to false: sends pearled players to free world if freed when offline. if set to true: keeps players in end if freed when offline.
 free_tppearl: true
+# if freed while online, will teleport to the player holding the pearl.
 damagelog_min: 3
+# unsure
 damagelog_ticks: 600
```

Comment:
The amount of ticks the damagelogger keeps track of damages done to players before resetting.